service/request/index.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+import got from 'got';
+import merge from 'lodash.merge';
+
```

Comment:
JSDoc?

Reply:
No default timeout: potential for hanging requests.
Consider a modest default (e.g. 10s) with override.